index.bs

```diff
@@ -52,43 +52,37 @@ consent of the user who registered the <a>scoped credential</a>. Functionally, t
 methods (along with associated data structures): <a>makeCredential()</a> and <a>getAssertion()</a>. The former is used during
 <a>Registration</a> and the latter during <a>Authentication</a>.
 
-
-Broadly, <a>authenticators</a> are of two types:
-
-1. <dfn>Embedded authenticators</dfn> have their operation managed by the same computing device (e.g., smart phone, tablet,
-    desktop PC) as the user agent is running on. For instance, such an authenticator might consist of a Trusted Execution
-    Environment (TEE) applet, a Trusted Platform Module (TPM), or a Secure Element (SE) integrated into the computing device in
-    conjunction with some means for <a>user verification</a>, along with appropriate platform software to mediate access to
-    these components' functionality.
-
-2. <dfn>External authenticators</dfn> operate autonomously from the computing device running the user agent, and are accessed
-    over a transport such as Universal Serial Bus (USB), Bluetooth Low Energy (BLE) or Near Field Communications (NFC). 
-
-Note that an external authenticator may itself contain an embedded authenticator. For example, consider a smart phone that
-contains a scoped credential. The credential may be accessed by a web browser running on the phone itself. In this case the
-module containing the credential is functioning as an embedded authenticator. However, the credential may also be accessed over
-BLE by a user agent on a nearby laptop. In this latter case, the phone is functioning as an external authenticator. These modes
-may even be used in a single end-to-end user scenario. One such scenario is described among the use cases below (in
-[[#authentication-external]]).
+Broadly, <a>authenticators</a> operate in different ways, but compliant
```

Comment:
Suggest deleting the words "<a>authenticators</a> operate in different ways, but" since they add little information content to the sentence.